galaxy.yml

```diff
@@ -2,7 +2,7 @@
 namespace: kubernetes_sigs
 description: Deploy a production ready Kubernetes cluster
 name: kubespray
-version: 2.23.1
+version: 2.24.0
 readme: README.md
```

Comment:
Hi @VannTen 

It maybe better to use another PR to modify.
Because the PR is for docs changes :-)